pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -73,10 +73,9 @@ func ConvertService(svc corev1.Service, domainSuffix string, clusterID cluster.I
 		addr = svc.Spec.ClusterIP
 		if len(svc.Spec.ClusterIPs) > 0 {
 			for _, ip := range svc.Spec.ClusterIPs {
-				// exclude the svc.Spec.ClusterIP
-				if ip != addr {
-					extrAddrs = append(extrAddrs, ip)
-				}
+				// pilot/pkg/serviceregistry/kube/controller/serviceimportcache.go also performs a conversion from k8s svc -> model
+				// it doesn't exclude the "default" IP from this slice, this conversion should align which also makes it more like k8s
+				extrAddrs = append(extrAddrs, ip)
```

Comment:
So we will have a list like `[]{ip1, ip1, ip2}`? Is that what we want? seems a bit odd..?

Reply:
100% it's not. I had wanted []{ip1, ip2} and had not seen that around line 110 when we build the struct we prepend addr to extraAddrs. TY